portfolio/src/main/webapp/script.js

```diff
@@ -76,9 +76,9 @@ function nextProj() {
 }
 
 function prevProj(){
-    projectBoxCounter-=2;
+    projectBoxCounter -= 2;
     if(projectBoxCounter== -2){
-        projectBoxCounter=7;
+        projectBoxCounter = 6;
```

Comment:
document magic number: prefer named constant

Reply:
okay, I will fix that, thanks for the suggestion!